### ISSUE TITLE
Provider path using `--provider` doesn't work on Windows

### ISSUE THREAD

**Issue by whip113** [cyberark/summon#87]
It would appear our hard coded value is making summon ignore what is provided with the `--provider` switch.

**Comment by sgnn7** [comment]
@whip113 Can you provide more details about how to recreate this bug? I have attempted to recreate this and was unable to do so locally.

**Comment by sgnn7** [comment]
@whip113 I will close this issue since I have not heard back from you but feel free to ping me if this is an issue in the latest releases.

**Comment by mdzieciol** [comment]
I have the same problem.
```
PS C:\Users\marcin.dzieciol> C:\Users\marcin.dzieciol\Downloads\summon.exe --yaml 'pass: !var pass' .\pass.ps1
Error fetching variable pass: exec: "C:\\Program Files/Cyberark Conjur/Summon/Providers/conjur": file does not exist
```

in this example I've tried to use second option from this document
`https://github.com/cyberark/summon/blob/master/provider/README.md`
Defining path as a user environment variable.
I think the problem is combining backslashes and slashes in one path. 

when I'm trying to provide provider path i have this result:
```
PS C:\Users\marcin.dzieciol> C:\Users\marcin.dzieciol\Downloads\summon.exe -p "C:\Program Files\Cyberark Conjur\Summon\Providers\conjur" --yaml 'pass: !var pass' .\pass.ps1
CreateFile C:\Program Files/Cyberark Conjur/Summon/Providers/C:\Program Files\Cyberark Conjur\Summon\Providers\conjur: The filename, directory name, or volume label syntax is incorrect.
```

it doesn't make sense, could you please fix this. For now version on windows is not usable.

**Comment by sgnn7** [comment]
@mdzieciol I think in theory should work fine.
Can you:
- Provide the `dir` listing of your `C:\Program Files\Cyberark Conjur\Summon\Providers` directory?
- Check that the provider filename ends in `.exe` extension?
- Check that the permissions on the provider file are correct? (what happens when you run `conjur` directly?)

**Comment by sgnn7** [comment]
Can you also make sure that Windows isn't preventing you from running this file? You may need to escape the spaces though and I am not on a Windows machine right now but try:
```powershell
dir "C:\Program Files\Cyberark Conjur\Summon\Providers" -Recurse | Unblock-File
```

**Comment by mdzieciol** [comment]
1. 
```
PS C:\Users\marcin.dzieciol\Downloads> dir "C:\Program Files\Cyberark Conjur\Summon\Providers"


    Directory: C:\Program Files\Cyberark Conjur\Summon\Providers


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        3/31/2020  10:20 PM            259 conjur

```
2.
But why it should be .exe extention. In this file I have configuration for Conjur. If this should be an exe, where I find I to download? And how to provide envirement variables like URL, user, api-key etc.?

3. the same as 2nd answare. To be honest, in readme I didn't find any information how to configure windows version. I think I did something wrong if you asking me for exe file.

**Comment by mdzieciol** [comment]
ah ok, I found it.
Can you tell me, how to provide variables?
I have this error:
'''
Failed creating a Conjur client: Must specify an ApplianceURL -- Must specify an Account
'''

**Comment by whip113** [comment]
@mdzieciol, could I ask you to post this question over on https://discuss.cyberarkcommons.org/ please? There is a broader audience there then in a closed Github issue and the community will benefit from the discussion.

**Comment by izgeri** [comment]
I've opened a new issue for @mdzieciol's issues in #167 - let's have any further conversation there.
